code/modules/projectiles/projectile/reusable.dm

```diff
@@ -57,7 +57,7 @@
 	newdart.update_icon()
 
 /obj/item/projectile/bullet/reusable/foam_dart/Destroy()
-	pen = null
+	QDEL_NULL(pen)
```

Comment:
This is OK since the pen will be deleted with the `contents` scour higher up the hierarchy